enterprise/internal/licensing/data.go

```diff
@@ -37,51 +37,62 @@ var allPlans = []Plan{
 	// OAuth, GitLab OAuth, SAML, and OpenID.
-	FeatureSSO Feature = "sso"
+	FeatureSSO BasicFeature = "sso"
 
 	// FeatureACLs is whether the Background Permissions Syncing may be be used for
 	// setting repository permissions.
-	FeatureACLs Feature = "acls"
+	FeatureACLs BasicFeature = "acls"
 
 	// FeatureExplicitPermissionsAPI is whether the Explicit Permissions API may be be used for
 	// setting repository permissions.
-	FeatureExplicitPermissionsAPI Feature = "explicit-permissions-api"
+	FeatureExplicitPermissionsAPI BasicFeature = "explicit-permissions-api"
 
 	// FeatureExtensionRegistry is whether publishing extensions to this Sourcegraph instance has been
 	// purchased. If not, then extensions must be published to Sourcegraph.com. All instances may use
 	// extensions published to Sourcegraph.com.
-	FeatureExtensionRegistry Feature = "private-extension-registry"
+	FeatureExtensionRegistry BasicFeature = "private-extension-registry"
 
 	// FeatureRemoteExtensionsAllowDisallow is whether explicitly specify a list of allowed remote
 	// extensions and prevent any other remote extensions from being used has been purchased. It
 	// does not apply to locally published extensions.
-	FeatureRemoteExtensionsAllowDisallow Feature = "remote-extensions-allow-disallow"
+	FeatureRemoteExtensionsAllowDisallow BasicFeature = "remote-extensions-allow-disallow"
 
 	// FeatureBranding is whether custom branding of this Sourcegraph instance has been purchased.
-	FeatureBranding Feature = "branding"
+	FeatureBranding BasicFeature = "branding"
 
 	// FeatureCampaigns is whether campaigns (now: batch changes) on this Sourcegraph instance has been purchased.
 	//
 	// DEPRECATED: See FeatureBatchChanges.
-	FeatureCampaigns Feature = "campaigns"
-
-	// FeatureBatchChanges is whether Batch Changes on this Sourcegraph instance has been purchased.
-	FeatureBatchChanges Feature = "batch-changes"
+	FeatureCampaigns BasicFeature = "campaigns"
 
 	// FeatureMonitoring is whether monitoring on this Sourcegraph instance has been purchased.
-	FeatureMonitoring Feature = "monitoring"
+	FeatureMonitoring BasicFeature = "monitoring"
 
 	// FeatureBackupAndRestore is whether builtin backup and restore on this Sourcegraph instance
 	// has been purchased.
-	FeatureBackupAndRestore Feature = "backup-and-restore"
+	FeatureBackupAndRestore BasicFeature = "backup-and-restore"
 
 	// FeatureCodeInsights is whether Code Insights on this Sourcegraph instance has been purchased.
-	FeatureCodeInsights Feature = "code-insights"
+	FeatureCodeInsights BasicFeature = "code-insights"
 )
 
+// FeatureBatchChanges is whether Batch Changes on this Sourcegraph instance has been purchased.
+type FeatureBatchChanges struct {
+	// If true, there is no limit to the number of changesets that can be created.
+	Unrestricted bool
+	// Maximum number of changesets that can be created. If Unrestricted is true, this is ignored.
+	MaxNumBatchChanges int
```

Comment:
```suggestion
	// Maximum number of changesets that can be created. If Unrestricted is true, this is ignored.
	MaxChangesets int
```

to match the comment. And also see my comment further above.